src/test/java/HttpServer/resource/FileResourceTest.java

```diff
@@ -0,0 +1,34 @@
+package HttpServer.definer;
+
+import HttpServer.resource.FileResource;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class FileResourceTest {
+
+    private FileResource resource;
+
+    @Before
+    public void setup() {
+        File directory = new File("./cob_spec/public");
+        File file = new File(directory.getAbsolutePath(), "text-file.txt");
+        resource = new FileResource(file);
+    }
+
```

Comment:
What happens if the file with the same name is in a different directory will this class still work?  
`./cob_spec/public/text-file.txt` and `./cob_spec/public/files/text-file.txt`  Might be something that should be tested..